bufferserver/src/main/java/com/datatorrent/bufferserver/internal/DataList.java

```diff
@@ -176,21 +176,20 @@ public void reset()
     numberOfInMemBlockPermits.set(MAX_COUNT_OF_INMEM_BLOCKS - 1);
   }
 
-  public void purge(final int baseSeconds, final int windowId)
```

Comment:
That was the suggestion by @vrozov to remove the extra API. I don't have any strong opinion about them.